src/theme.js

```diff
@@ -0,0 +1,17 @@
+export const theme = {
+  color: {
+    whisper: "#F5F5FA",
+    white: "#ffffff",
+    lightBlue: "#D6E4FF",
+    active: "#0044CC",
+    grey: "E4E6F0",
+    darkerGrey: "#7E839A",
+    stormGrey: "#74788B",
+    inactive: "#7E839A",
+    waterloo: "#7E839A",
```

Comment:
Great job ! 👌👌

In my opinion active and inactive aren't good name for color. It would be better to change. And these (higlighted) colors are duplicated also, however - Great job!

Reply:
You are right, this name is not appropriate, I already changed it and added 2 more colors.